packages/mermaid/src/diagrams/git/gitGraphAst.ts

```diff
@@ -57,8 +61,8 @@ function getId() {
 // }
 
 /**
- * @param currentCommit
- * @param otherCommit
+ * @param currentCommit - current commit
+ * @param otherCommit - other commit
```

Comment:
nit: Feel free to remove comments like this, if they're not adding any additional value.